easybuild/easyconfigs/r/RELION/RELION-2.1-foss-2018a.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '2.1'
+
+homepage = 'http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page'
+description = """RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer
+ program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class
+ averages in electron cryo-microscopy (cryo-EM)."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'opt': True}
+
+source_urls = ['https://github.com/3dem/relion/archive']
+sources = ['%s.tar.gz' % version]
+checksums = ['08729aac4fa0841e08df2c90a2c26cfe2aef52af2f5fb94a770ed2c11373b886']
+
+builddependencies = [('CMake', '3.10.1')]
+
+dependencies = [
+    ('X11', '20180131'),
+    ('FLTK', '1.3.4'),
+]
+
+configopts = "-DCMAKE_SHARED_LINKER_FLAGS='-lpthread' "
+
```

Comment:
@smoors Please also include a custom `sanity_check_paths` (the fact that the existing easyconfigs don't have this is a bug).